### PR TITLE
Bevy Foundation: announcement and foundation pages

### PR DESCRIPTION
**Pull request by cart**
This makes a variety of changes:

1. Adds a blog post announcing the Bevy Foundation!
2. Adds a new, config-driven donation system, which will be used to fund the Bevy Foundation
3. Adds Bevy Foundation pages: a Foundation home page, meeting minutes, bylaws, etc
4. Updates our Sponsorship Pledge to account for the existence of the Bevy Foundation

The maintainers (now board directors!) and I have already reviewed this, so the review cycle on this will be short.

### CODE REVIEW

**Review by alice-i-cecile — APPROVED** [review]
Did another pass: I'm happy with this :)

**Review by james7132 — APPROVED** [review]

**Review by superdump — APPROVED** [review]

**Review by mockersf — APPROVED** [review]

File: content/news/2024-03-11-bevy-foundation/index.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+youtube = "cartdev"
++++
+
+It is with excitement that I unveil to the world ...
+
+<img src="bevy_foundation.svg" alt="Bevy Foundation Logo" style="margin-top: 2rem; margin-bottom: 2rem;" />
+
+The [**Bevy Foundation**](/foundation) is the next step in our journey to build a world class free and open source game engine. **Bevy Foundation** is a non-profit organization formed in Washington State, with a pending federal 501(c)(3) tax-exemption application (more on this later).
+
+<!-- more -->
```
Comment (IceSentry) [inline content/news/2024-03-11-bevy-foundation/index.md:19]: Is this here because you want to add more content or is it just a dead comment?

Comment (cart) [reply content/news/2024-03-11-bevy-foundation/index.md:19]: This is a Special Zola Thing that determines the cutoff for the description in `/news` card for this post

File: content/foundation/bylaws/index.md [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+template = "news-page.html"
+date = 2024-02-03
+[extra]
+author = "Bevy Foundation Board of Directors"
+header_message = "Foundation Bylaws"
++++
+
+## Article 1. Offices
+
+The principal office of Bevy Foundation (the “Corporation) shall be located at its principal place of business or such other place as the Board of Directors (“Board”) may designate. The Corporation may have other such offices, either within or without the State of Washington, as the Board may designate or as the business of the Corporation may require from time to time.
```
Comment (rparrett) [inline content/foundation/bylaws/index.md:12]:
```suggestion
The principal office of Bevy Foundation (the “Corporation") shall be located at its principal place of business or such other place as the Board of Directors (“Board”) may designate. The Corporation may have other such offices, either within or without the State of Washington, as the Board may designate or as the business of the Corporation may require from time to time.
```

**Review by matiqo15 — APPROVED** [review]

File: content/foundation/bylaws/index.md [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+template = "news-page.html"
+date = 2024-02-03
+[extra]
+author = "Bevy Foundation Board of Directors"
+header_message = "Foundation Bylaws"
++++
+
+## Article 1. Offices
+
+The principal office of Bevy Foundation (the “Corporation") shall be located at its principal place of business or such other place as the Board of Directors (“Board”) may designate. The Corporation may have other such offices, either within or without the State of Washington, as the Board may designate or as the business of the Corporation may require from time to time.
```
Comment (rparrett) [inline content/foundation/bylaws/index.md:12]:
```suggestion
The principal office of Bevy Foundation (the “Corporation”) shall be located at its principal place of business or such other place as the Board of Directors (“Board”) may designate. The Corporation may have other such offices, either within or without the State of Washington, as the Board may designate or as the business of the Corporation may require from time to time.
```

Sorry, I suggested a non-matching double quote character and you clicked the button before I managed to edit in the right one.

**Pull request merged**